iceoryx_posh/test/moduletests/test_mepoo_segment.cpp

```diff
@@ -152,8 +148,13 @@ class MePooSegment_test : public Test
     iox::posix::Allocator m_managementAllocator{iox::posix::Allocator(m_rawMemory, RawMemorySize)};
 
     MePooConfig mepooConfig = setupMepooConfig();
-    MePooSegment<SharedMemoryObject_MOCK, MemoryManager> sut{
-        mepooConfig, m_managementAllocator, PosixGroup{"iox_roudi_test1"}, PosixGroup{"iox_roudi_test2"}};
+
+    using SUT = MePooSegment<SharedMemoryObject_MOCK, MemoryManager>;
+    std::unique_ptr<SUT> createSut()
+    {
+        return std::make_unique<SUT>(
+            mepooConfig, m_managementAllocator, PosixGroup{"iox_roudi_test1"}, PosixGroup{"iox_roudi_test2"});
+    }
```

Comment:
I had to delay the creation of the SUT since it caused test failures when the required user do not exist